Library/Formula/rpm.rb

```diff
@@ -52,18 +56,20 @@ def install
         --with-neon=external
         --with-uuid=external
         --with-pcre=external
-        --with-lua=internal
-        --with-syck=internal
+        --with-lua=external
+        --with-syck=external
```

Comment:
Basically anything that ships with OS X let's use the OS X version (except OpenSSL if that's required) and let's use the static/internal libraries. Want to avoid additional Homebrew dependencies when we can.